wiki/en/Running-a-Server.md

```diff
@@ -92,16 +92,14 @@ If you want to run a Server on a Raspberry Pi, you will need to [compile from so
 
 ### Installation
 
-1. Download the [latest headless .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless }})
+1. Download the latest headless .deb file: `wget {{ site.download_root_link }}{{ site.download_file_names.deb-headless }}`
 1. Update apt to make sure you have a current list of standard packages: `sudo apt update`
 1. Install the Jamulus package: `sudo apt install ./{{ site.download_file_names.deb-headless }}`
 1. Enable the headless Server process via systemd: `sudo systemctl enable jamulus-headless`
-1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file in `/lib/systemd/system/jamulus-headless.service` (By default you will be running an Unregistered Server).
+1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file by running `systemctl edit jamulus-headless` (By default you will be running an Unregistered Server).
```

Comment:
```suggestion
1. Add your desired [command line options](Running-a-Server#command-line-options) to the `ExecStart` line in the systemd service file by running `sudo systemctl edit --full jamulus-headless` (By default you will be running an Unregistered Server).
```

Reply:
That works for me. Don't need to `sudo systemctl daemon-reload` either after you do the edit either as far as I can tell. Just edit, `systemctl restart jamulus-headless` and it's fine.